concrete_biopython/SeqWrapper.py

```diff
@@ -22,7 +22,7 @@ class SeqWrapper():
     """
 
     # All letters of the default alphabet
-    _DEFAULT_ALPHABET = '\0*ABCDEFGHIJKLMNOPQRSTUVWXYZ'
+    _DEFAULT_ALPHABET = '\0!\"#$%&\'()*+,-./0123456789:;<=>?@ABCDEFGHIJKLMNOPQRSTUVWXYZ[\\]^_`abcdefghijklmnopqrstuvwxyz{|}~\t\n'
```

Comment:
What's the purpose of this change?

Reply:
to be able to use all ASCII character by default

Reply:
But isn't it slower? I think we should always force the users to specify the alphabet.

Reply:
ok I will enforce it